sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -407,13 +408,14 @@ public partial class ServiceBusSessionProcessor : System.IAsyncDisposable
         public virtual System.Threading.Tasks.Task StopProcessingAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override string ToString() { throw null; }
-        public void UpdateConcurrency(int maxConcurrentSessions, int maxConcurrentCallsPerSession) { }
+        public void UpdateConcurrency(int maxConcurrentSessions, int maxConcurrentCallsPerSession, int? maxConcurrentCallsAcrossAllSessions = default(int?)) { }
```

Comment:
Might be worth throwing if this value is greater than maxConcurrentSessions*maxConcurrentCallsPerSession.